### PR TITLE
QEMU: Fix issue 72191

### LINKED ISSUE #72191: qemu -no-acpi command removed in 9.0
running west command:

- west build -b qemu_x86 samples/hello_world
- west build -t run

error occurs stating -no-acpi is not a command.
It was removed in qemu 9.0, see https://www.qemu.org/docs/master/about/removed-features.html
Stated in file: zephyr/boards/qemu/x86/board.cmake

Removing -no-acpi works, tried like this:
/usr/local/bin/qemu-system-i386 -m 32 -cpu qemu32,+nx,+pae -machine q35 -device isa-debug-exit,iobase=0xf4,iosize=0x04 -no-reboot -nographic -net none -pidfile qemu.pid -chardev stdio,id=con,mux=on -serial chardev:con -mon chardev=con,mode=readline -icount shift=5,align=off,sleep=off -rtc clock=vm -kernel /$HOME/zephyrproject/zephyr/build/zephyr/zephyr.elf

### PR DESCRIPTION
**Pull request by glenn-andrews**
Fixes https://github.com/zephyrproject-rtos/zephyr/issues/72191 by replacing `-no-acpi` with `-machine acpi=off`

Appears to still work with QEMU pre 9.0.0

### CODE REVIEW

**Review by jhedberg — APPROVED** [review]

**Review by tomi-font — APPROVED** [review]
Not sure how far back dates the `-machine acpi=off` option, but I think it's still okay to unconditionally use it.

**Comment by fabiobaltieri** [comment]
@glenn-andrews could you use a more meaningful description for the PR commit message? The first line should describe what is changed, the commit message should the describe why is changed. The issue number reference is meaningless once the project is forked, I would suggest dropping it and keeping it in the PR comment (not the commit message), if you really want to keep it in the commit message at least refer to full URL, maybe you can use the `Link:` tag.

See https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines for mode details.

**Review by fabiobaltieri — DISMISSED** [review]
(see the comment above)

**Review by kartben — APPROVED** [review]

**Comment by fabiobaltieri** [comment]
> Better?

Yeah, though it should be lowercase and no dot at the end for coherency (see `git log --oneline boards`). Good enough though but feel free to tweak it further, waiting for a CI run anyway before merging more

**Comment by fabiobaltieri** [comment]
> waiting for a CI run anyway before merging more

yeah nevermind, that'll do :-)

**Pull request merged**
